src/screens/CreatePost/index.js

```diff
@@ -247,8 +247,8 @@ const CreatePost = () => {
     }
   ];
 
-  const handleLocation = async (res) => {
-    setGeoList([...location, ...res?.locations]);
+  const handleLocation = async () => {
+    setGeoList([...location]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in `handleLocation` function seems to remove the ability to add new locations from the response (`res`) to the `GeoList`. If this is intentional and you only want to set `GeoList` with the existing `location` array, then it's fine. However, if you still need to add new locations from the response, you should revert this change.
```diff
-   const handleLocation = async () => {
-     setGeoList([...location]);
+   const handleLocation = async (res) => {
+     setGeoList([...location, ...res?.locations]);
     setLoading(false);
   };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->